I0328 23:52:40.352708       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Role/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:52:40.356505       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ServiceAccount/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:52:40.367450       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create RoleBinding/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:52:40.372100       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Deployment/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:52:40.377603       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Service/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:52:40.382655       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd in  test-cluster successful
I0328 23:52:40.385825       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd-peer in  test-cluster successful
I0328 23:52:40.389668       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-pd-3731616 for controller TidbCluster/test-cluster successfully
I0328 23:52:40.395527       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"24813", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-pd in  test-cluster successful
I0328 23:52:40.406163       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:52:40.406199       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
I0328 23:52:40.530886       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:52:40.530923       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
I0328 23:52:40.624747       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:52:41.469143       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
I0328 23:52:41.493440       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:52:41.493502       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:52:41.553747       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
E0328 23:52:41.562219       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:52:49.559685       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
E0328 23:52:49.569443       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:52:55.372770       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused, service acto-namespace/test-cluster-pd has no endpoints
I0328 23:52:55.409165       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:52:55.409202       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:52:56.492781       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused, service acto-namespace/test-cluster-pd has no endpoints
I0328 23:52:56.505322       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:53:02.224838       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
I0328 23:53:02.254708       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:02.254747       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:53:02.317131       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
I0328 23:53:02.371120       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:02.371163       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:53:02.435884       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
I0328 23:53:02.487528       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:02.487569       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:53:02.547659       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
I0328 23:53:02.556832       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:53:02.967060       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
I0328 23:53:02.979838       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25082", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tikv-peer in  test-cluster successful
I0328 23:53:02.982987       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25082", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tikv-3831336 for controller TidbCluster/test-cluster successfully
I0328 23:53:02.989714       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25082", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tikv in  test-cluster successful
I0328 23:53:03.005212       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:03.005253       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
E0328 23:53:03.310920       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
I0328 23:53:03.362064       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:53:03.362122       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.170.108:2379: connect: connection refused, requeuing
E0328 23:53:03.912684       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
I0328 23:53:03.962344       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:53:03.962404       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.170.108:2379: connect: connection refused, requeuing
E0328 23:53:04.510917       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.170.108:2379: connect: connection refused
E0328 23:53:04.522057       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.170.108:2379: connect: connection refused, requeuing
E0328 23:53:15.221385       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
E0328 23:53:15.235350       1 tikv_upgrader.go:249] tikv: no store found for TiKV ordinal 0 of acto-namespace/test-cluster
I0328 23:53:15.253500       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:53:15.253553       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores,body response: no leader
, requeuing
I0328 23:53:15.334354       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25239", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-2(4086175127003262415) is unhealthy
I0328 23:53:15.334485       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25239", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0328 23:53:15.350971       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:53:15.351010       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 1 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
I0328 23:53:15.434084       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25244", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-2(4086175127003262415) is unhealthy
I0328 23:53:15.434176       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25244", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
E0328 23:53:15.434118       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 1 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
E0328 23:53:21.657536       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0328 23:53:21.657626       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25244", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0328 23:53:21.692212       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:21.692242       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
E0328 23:53:21.770855       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0328 23:53:21.771045       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25270", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0328 23:53:21.786209       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0328 23:53:24.370607       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25270", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-2(4086175127003262415) is unhealthy
I0328 23:53:24.370691       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25270", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0328 23:53:24.391005       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:53:24.391050       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 1 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
I0328 23:53:24.471357       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25284", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-2(4086175127003262415) is unhealthy
E0328 23:53:24.471402       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 1 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
I0328 23:53:24.471447       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25284", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0328 23:53:54.397251       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25284", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb-peer in  test-cluster successful
I0328 23:53:54.409650       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25284", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb in  test-cluster successful
I0328 23:53:54.413565       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25284", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tidb-6662316 for controller TidbCluster/test-cluster successfully
I0328 23:53:54.420762       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"21beec5a-0a68-467e-a548-11ae2db4706c", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"25284", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tidb in  test-cluster successful
I0328 23:53:55.228460       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:55.316773       1 equality.go:87] Service spec diff for acto-namespace/test-cluster-tidb:   v1.ServiceSpec{
  	Ports: []v1.ServicePort{
  		{
  			... // 3 identical fields
  			Port:       4000,
  			TargetPort: {IntVal: 4000},
- 			NodePort:   0,
+ 			NodePort:   31566,
  		},
  		{
  			... // 3 identical fields
  			Port:       10080,
  			TargetPort: {IntVal: 10080},
- 			NodePort:   0,
+ 			NodePort:   30588,
  		},
  	},
  	Selector:  {"app.kubernetes.io/component": "tidb", "app.kubernetes.io/instance": "test-cluster", "app.kubernetes.io/managed-by": "tidb-operator", "app.kubernetes.io/name": "tidb-cluster"},
  	ClusterIP: "",
  	... // 12 identical fields
  }
I0328 23:53:55.316802       1 tidb_member_manager.go:442] Sync TiDB service acto-namespace/test-cluster-tidb, spec equal: false, annotations equal: true, label equal: true
I0328 23:53:55.600839       1 service_control.go:88] update Service: [acto-namespace/test-cluster-tidb] successfully, kind: , name: test-cluster
I0328 23:53:55.623675       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
E0328 23:53:56.202344       1 pod_control.go:185] failed to update pod acto-namespace/test-cluster-tidb-2 with cluster labels map[app.kubernetes.io/component:tidb app.kubernetes.io/instance:test-cluster app.kubernetes.io/managed-by:tidb-operator app.kubernetes.io/name:tidb-cluster controller-revision-hash:test-cluster-tidb-74b4fb8dbc statefulset.kubernetes.io/pod-name:test-cluster-tidb-2 tidb.pingcap.com/cluster-id:7215726660903715091], TidbCluster: test-cluster, err: Operation cannot be fulfilled on pods "test-cluster-tidb-2": the object has been modified; please apply your changes to the latest version and try again
I0328 23:53:56.450699       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:56.563781       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:53:56.614073       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:53:56.974769       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:53:58.173993       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:54:24.410351       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:54:54.395035       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:55:01.815565       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:55:24.398364       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:55:54.417194       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:56:24.415408       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:56:54.417724       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:57:24.412609       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:57:54.413255       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:58:24.409407       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:58:54.404966       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:59:24.418838       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:59:54.414611       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0329 00:00:24.413879       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0329 00:00:54.419151       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0329 00:01:24.409261       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0329 00:01:54.418689       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0329 00:02:24.408785       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped

